examples/globe_geojson_to3D.html

```diff
@@ -62,13 +62,15 @@
             ariege.convert = itowns.Feature2Mesh.convert({
                 color: color,
                 extrude: extrude });
+            ariege.projection = view.tileLayer.extent.crs();
 
             ariege.source = new itowns.FileSource({
                 url: 'https://raw.githubusercontent.com/gregoiredavid/france-geojson/master/departements/09-ariege/departement-09-ariege.geojson',
                 projection: 'EPSG:4326',
                 format: 'application/json',
                 zoom: { min: 7, max: 7 },
-            }, view.referenceCrs);
+                parser: itowns.GeoJsonParser.parse,
```

Comment:
it will be necessary to make more friendly: add specofic `Source` by format. This is **Breaking change**? it's necessary to add parser?

Reply:
Yes I agree, we can add some new Sources. This is not a breaking change, as `format` is still read. In fact, we don't really need to have the parser specified here.

Reply:
Removed to avoid the confusion